src/main/scala/com/codacy/client/bitbucket/v2/service/RepositoryServices.scala

```diff
@@ -12,11 +12,12 @@ class RepositoryServices(client: BitbucketClient) {
    * if the caller is authenticated and is authorized to view the repository.
    */
   def getRepositories(
-      ownerInfo: OwnerInfo,
+      ownerInfo: Option[OwnerInfo],
       pageLength: Option[Int] = Option(100),
       userRole: Option[Role] = None
   ): RequestResponse[Seq[Repository]] = {
-    val baseUrl = s"https://bitbucket.org/api/2.0/repositories/${ownerInfo.value}"
+    val owner = ownerInfo.fold("")(info => info.value)
+    val baseUrl = s"https://bitbucket.org/api/2.0/repositories/$owner"
```

Comment:
Please test this well with the pagination. Because the https://bitbucket.org/api/2.0/repositories/ and 
https://bitbucket.org/api/2.0/repositories/ use different pagination techniques